quactography/visu/gs_square_loss_for_p.py

```diff
@@ -84,39 +86,42 @@ def visualize_optimal_paths_edge_alpha(
     Returns
     -------
     None """
-    reps = []
-    alphas = []
-    square_loss = []
-    param_count = 0
+    delta_dict = defaultdict(list)
     path = Path(in_folder)
 
     glob_path = path.glob('*.npz')
 
     for in_file_path in glob_path:
-        _, _, min_cost, h, _, rep, _ = load_optimization_results(in_file_path)
+        _, dist_prob, min_cost, h, _, _, _ = load_optimization_results(in_file_path)
         min_cost = min_cost.item()
+        dist_prob = dist_prob.item()
         h = h.item()
-        alpha = h.alpha * h.graph.number_of_edges/h.graph.all_weights_sum
-        alphas.append(alpha)
-
-        if rep not in reps:
-            param_count += 1
+        alpha = h.alphai

```

Comment:
Are all the alphas interchangeable, why not `alphai = h.alphai` (or a better name)